clouddriver-elasticsearch-aws/src/main/kotlin/com/netflix/spinnaker/clouddriver/elasticsearch/aws/ElasticSearchAmazonCachingAgentProvider.kt

```diff
@@ -67,6 +70,15 @@ open class ElasticSearchAmazonCachingAgentProvider(
         amazonClientProvider,
         credentials,
         elasticSearchClient
+      ),
+      ClusterScalingPolicyTaggingAgent(
```

Comment:
I would probably jam this agent into a different provider as the rest of the elastic search instance and server group caching agents are currently disabled in production (and we should be able to enable the scaling policy tagging independently)